<launch>
  <param name="/camera_topic" type="string" value="/image_raw" />
  <node name="HoG" pkg="hog" type="body_detector" output="screen"/>
</launch>